packages/core/src/__tests__/server-side-hydrate.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import ReactDOM from 'react-dom';
+import { styled } from '@compiled/core';
+
+describe('server side hydrate', () => {
+  it('should ssr then hydrate element correctly', () => {
```

Comment:
could we rename this to `it should not log unexpected warnings when hydrating SSRing`